### PR TITLE
Translate live label

### LINKED ISSUE #5749: Show Turkish & Vietnamese translation for Live label
**Is your feature request related to a problem? Please describe.**
Use `CANLI` instead of `LIVE` for turkish live label. 
Use `TRỰC TIẾP` instead of `LIVE` for vietnamese live label. 
Use `لائیو` instead of `LIVE` for urdu live label. 

**Describe the solution you'd like**
Translation is already in the Simorgh codebase. We should ensure the Live Label is using this translation in the Story Promo:

https://github.com/bbc/simorgh/blob/latest/src/app/containers/StoryPromo/index.jsx#L83-L86

We should ensure the `VisuallyHiddenText lang="en-GB"` element is not added if text is not in English.

We should also check that the live label is added in all relevant places, e.g. Bulletins (Radio/TV), Radio schedules.

- In Bulletins, the Live label will always be aria-hidden, with visually hidden text that reads 'Watch Live,', with watch translated into the correct language. This will be updated to contain a fully translated string in another issue.

**Describe alternatives you've considered**

**Testing notes**

Dev insight: Unit tests/snapshots should be sufficient. Adding screenshots of the changes made to the PR would be great. 

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by tochwill**
Resolves #5749

**Overall change:** 
This pulls in the newest version of psammead story promo - it uses the translation of 'Live' from the translation bundles for all services, and only uses visually hidden text if the english label is used.

**Code changes:**
- This pulls in the newest version of psammead story promo
- Uses the translation of 'Live' from the translation bundles for all services
- Only uses visually hidden text if the english label is used.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

Screenshots of new behaviour:

English markup:
image

Pidgin markup:
Screenshot 2020-03-04 at 14 46 39

Pidgin story promo:
Screenshot 2020-03-04 at 14 46 14

English story promo:
Screenshot 2020-03-04 at 14 46 06

Igbo radio bulletin:
Screenshot 2020-03-04 at 14 45 57

Urdu story promo:
Screenshot 2020-03-04 at 14 52 07

### CODE REVIEW

**Comment by tochwill** [comment]
This will fail until https://github.com/bbc/psammead/pull/3202 is published

File: src/app/containers/StoryPromo/index.jsx [hunk 1]
```diff
@@ -101,9 +119,17 @@ const StoryPromoContainer = ({
...
+  const {
+    script,
+    datetimeLocale,
+    service,
+    translations,
+    timezone,
+  } = useContext(ServiceContext);
+
+  const liveLabel = pathOr('LIVE', ['media', 'liveLabel'], translations);
+  const hiddenLiveLabel = liveLabel === 'LIVE' ? 'Live' : null;
```
Comment (sareh) [inline src/app/containers/StoryPromo/index.jsx]: I'd rather we have a `liveLabelIsEnglish` boolean which is passed into the LiveComponent and there we add the `<VisuallyHiddenText lang="en-GB">` Live, `</VisuallyHiddenText>` if this boolean is false and add the `aria-hidden: true` on the visible label in this case.

File: src/app/containers/StoryPromo/index.jsx [hunk 2]
```diff
@@ -77,21 +77,39 @@ StoryPromoImage.defaultProps = {
...
-  </span>
-);
+const LiveComponent = ({
+  headline,
+  service,
+  liveLabel,
+  hiddenLiveLabel,
+  dir,
+}) => {
+  const ariaHidden = hiddenLiveLabel && { 'aria-hidden': 'true' };
```
Comment (sareh) [inline src/app/containers/StoryPromo/index.jsx]: It'd be good to add a comment here explaining why we're conditionally adding an aria hidden to the visible live label text.

File: src/app/containers/Bulletin/index.test.jsx [hunk 3]
```diff
@@ -96,9 +96,10 @@ describe('Bulletin Container', () => {
       const { container } = render(BulletinWithContext(liveTvBulletinItem));
       const span = container.getElementsByTagName('span')[2];
 
-      expect(span.getAttribute('aria-hidden')).toBeDefined();
-      expect(span.getAttribute('aria-hidden')).toEqual('true');
-      expect(span.textContent).toEqual('LIVE');
+      expect(span.textContent).toEqual('NA EME UGBU A');
+      if (span.textContent === 'LIVE') {
```
Comment (tochwill) [inline src/app/containers/Bulletin/index.test.jsx]: I don't think this will test anything? We can add another test to do this though

Comment (tochwill) [reply src/app/containers/Bulletin/index.test.jsx]: This actually won't happen at the moment in Bulletin - we have a seperate ticket to amend how the visually hidden text works in here.

**Review by thekp — APPROVED** [review]
Looks good to me. 

My only thought would be to update psammead-bulletin here: https://github.com/bbc/psammead/blob/latest/packages/components/psammead-bulletin/src/index.jsx#L209-L211
but then I remembered we have plans to create a live label component.

File: src/app/containers/StoryPromo/index.jsx [hunk 4]
```diff
@@ -77,21 +77,30 @@ StoryPromoImage.defaultProps = {
...
-    <LiveLabel service={service} dir={dir}>
-      LIVE
-    </LiveLabel>
-    <VisuallyHiddenText lang="en-GB">{' Live, '}</VisuallyHiddenText>
-    {headline}
-  </span>
-);
+const LiveComponent = ({ headline, service, liveLabel, dir }) => {
+  // As screenreaders mispronounce the word 'LIVE', we use visually hidden
+  // text to read 'Live' instead, which screenreaders pronounce correctly.
```
Comment (sareh) [inline src/app/containers/StoryPromo/index.jsx:82]: Thanks for adding this explanation. We can reuse this when we pull it out into a psammead component.

**Review by sareh — APPROVED** [review]
Looks good. Thanks for the fix.

**Comment by PriyaKR** [comment]
Testing:

For three services Urdu,Turkce and vietnamese the radio bulletin live stories in storybook seems to pick up the correct translation and also the storypromo on frontpages.Refer attached screenshot.

Screen Shot 2020-03-04 at 15 32 12
Screen Shot 2020-03-04 at 15 32 48

For some reasons the TV bulletin live story on strorybook appears to be broken so not able to verify it.

**Comment by tochwill** [comment]
Thanks @PriyaKR - there doesn't seem to be any fixture data for a live TV bulletin.

**Comment by PriyaKR** [comment]
Live label on TV bulletin looks good on urdu frontpage.
Screen Shot 2020-03-04 at 15 58 33

**Pull request merged**